### ISSUE TITLE
Environments specifying older versions of Python are created with newer versions of python

### ISSUE THREAD

**Issue by anlutro** [tox-dev/tox#932]
Given a project with a `tox.ini` that contains something like this:

```
[tox]
envlist = py33, py34, py35, py36
```

I'm on a system with python 3.5, 3.6, and 3.7 installed. I would expect py33 and p34 to fail with "missing interpreter" or whatever. However, that's not what happens:

```
  py33: commands succeeded
SKIPPED:  py34: InterpreterNotFound: python3.4
  py35: commands succeeded
  py36: commands succeeded
```

```
$ ./.tox/py33/bin/python --version
Python 3.6.4
```

The same problem occurs with environments `py32` and `py31`, and I wouldn't be surprised if `py25` would use python 2.7 either (but I can't test that myself).

Tox is installed from pip in an isolated virtualenv:

```
$ tox --version
3.1.2 imported from /home/andreas/.local/share/psm/tox/lib/python3.6/site-packages/tox/__init__.py
```

This bug has already been fixed in https://github.com/tox-dev/tox/pull/914 but that fix is not in a current pypi release.

**Comment by asottile** [comment]
That fix is marked for the 3.2 milestone -- when 3.2 gets released it will work :)

**Comment by gaborbernat** [comment]
@asottile we'll keep it around so we release as bugfix now

**Comment by gaborbernat** [comment]
@anlutro released 3.1.3

**Comment by anlutro** [comment]
Cool, thanks!
